README.md

```diff
@@ -227,35 +227,19 @@ heat_configs:
   
 shift_stack: true
 ```
-2) Set ``ocp_base_domain`` in ``vars/shift_stack_vars.yaml`` to whatever domain you mentioned as ``NeutronDnsDomain`` in ``group_vars/all.yml``.  
-3) Set other parameters in ``vars/shift_stack_vars.yaml`` as per your specific requirement.  
-4) If you do not have an Openstack environment already set up, trigger deployment for Openstack along with Openshift, by running ``ansible-playbook main.yml``.  
-5) If you already have an existing Openstack environment on which you want to deploy Openshift using Jetpack, follow the steps mentioned below.  
-i) Prepare a hosts file like the one below, and save the file under the name ``hosts``.  
-```
-[undercloud]
-<IP address or DNS of undercloud host>
-
-[all:vars]
-ansible_connection=ssh
-ansible_user=stack
-ansible_ssh_pass=<password for stack user>
-```
-ii) Run ``ansible-playbook -i hosts -vvv ocp_on_osp.yml``  
-
-Cleanup of single Openshift deployment :  
-To destroy a single Openshift deployment, follow the steps mentioned below.  
-1) If you didn't already create a hosts file during deployment of Openshift on Openstack, create a hosts file like the one below, and save the file under the name ``hosts``.  
-```
-[undercloud]
-<IP address or DNS of undercloud host>
-
-[all:vars]
-ansible_connection=ssh
-ansible_user=stack
-ansible_ssh_pass=<password for stack user>
-```
-2) Run ``ansible-playbook -i hosts -vvv delete_single_ocp.yml``  
+2) Set ``ocp_base_domain`` in ``vars/shift_stack_vars.yaml`` to whatever domain you mentioned as ``NeutronDnsDomain`` in ``group_vars/all.yml``. 
+3) Set ``undercloud_host`` and ``undercloud_password`` in ``vars/shift_stack_vars.yaml`` 
+4) For installing multiple OpenShift clusters, increase ``ocp_cluster_count`` to desired count in ``vars/shift_stack_vars.yaml``
+4) Set other parameters in ``vars/shift_stack_vars.yaml`` as per your specific requirement.  
+5) If you do not have an Openstack environment already set up, trigger deployment for Openstack along with Openshift, by running ``ansible-playbook main.yml``.  
+6) If you already have an existing Openstack environment on which you want to deploy Openshift using Jetpack, run ``ansible-playbook -vvv ocp_on_osp.yml``  
```

Comment:
What if the infrared hosts file is not there, or has changed since then? Then, the user has to create the hosts file manually, right?

Reply:
Okay please ignore this comment. I guess this will use undercloud_host and undercloud_password in vars/shift_stack_vars.yaml.

Reply:
ack